### PR TITLE
Well-known service names come from the IANA list

### LINKED ISSUE #1296: Section 6.1: per service, per IP address
From the review by Roman Danyliw:

** Section 6.1.
-- Per “Port (a 16-bit integer):”, shouldn’t this be “16-bit unsigned integer”?
[ Note from @mwelzl: Erik Kline also suggested that in his review ]

-- Per “Service (an identifier that maps to a port; either the name of a
well-known service, or a DNS SRV service name to be resolved):”, what makes
something a “well-known service”?  Is this string have to be from
https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?

-- Per “IP address (IPv4 or IPv6 address):”, is there a formal syntax for an
IPv4 or IPv6 address that can be cited?

### PR DESCRIPTION
**Pull request by mwelzl**
Closes #1296

### CODE REVIEW

**Review by gorryfair — COMMENTED** [review]
Is this better as a REF?

**Comment by mwelzl** [comment]
How?  If there's a nice & clean way to do this, note that we have one more such long-URL IANA reference, probably best to make this uniform:
```
Namespaces for each of the keywords provided in the IANA protocol numbers registry (see https://www.iana.org/assignments/protocol-numbers/protocol-numbers.xhtml) are reserved for Protocol-specific Properties and MUST NOT be used for vendor or implementation-specific properties.
```

**Review by britram — APPROVED** [review]

File: draft-ietf-taps-interface.md [hunk 1]
```diff
@@ -735,7 +735,8 @@ RemoteSpecifier.WithHostName("example.com")
 RemoteSpecifier.WithPort(443)
 ~~~
 
-- Service (an identifier string that maps to a port; either the name of a well-known service, or a DNS SRV service name to be resolved):
+- Service (an identifier string that maps to a port; either a service
+name associated with a port number on https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml or a DNS SRV service name to be resolved):
```
Comment (tfpauly) [inline draft-ietf-taps-interface.md:739]:
```suggestion
name associated with a port number (from https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml) or a DNS SRV service name to be resolved):
```

File: draft-ietf-taps-interface.md [hunk 2]
```diff
@@ -735,7 +735,8 @@ RemoteSpecifier.WithHostName("example.com")
 RemoteSpecifier.WithPort(443)
 ~~~
 
-- Service (an identifier string that maps to a port; either the name of a well-known service, or a DNS SRV service name to be resolved):
+- Service (an identifier string that maps to a port; either a service
+name associated with a port number (from https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml) or a DNS SRV service name to be resolved):
```
Comment (tfpauly) [inline draft-ietf-taps-interface.md:739]:
```suggestion
name associated with a port number, from https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml, or a DNS SRV service name to be resolved):
```

**Pull request merged**